dipy/workflows/tests/test_reconst_mapmri.py

```diff
@@ -47,39 +45,39 @@ def reconst_mmri_core(flow, lap, pos):
 
         rtop = mmri_flow.last_generated_outputs['out_rtop']
         rtop_data = nib.load(rtop).get_data()
-        eq_(rtop_data.shape, volume.shape[:-1])
+        assert_equal(rtop_data.shape, volume.shape[:-1])
```

Comment:
If you want consistent style, you might go with `npt.assert_equal` here, as you had in other files (I think that's better -- more explicit).

Reply:
I agree,

I will update that